### DISCUSSION TITLE
Tailwind style not loaded in the stories

### DISCUSSION THREAD

**Discussion by LouisAndrew** [histoire-dev/histoire#262] [General]
First of all, thank you for Histoire, this tool is awesome 🎉 
I have a question on how to load tailwind style in the stories. The tailwind token on the design system works wonderfully, but I can't seem to be able to use the classes on the stories itself. Maybe there's something that I miss here?

Thank you!

**Comment by Akryum** [comment]
You need to import a CSS file with the 3 `@tailwind` directives in a setup file that you can specify with [`setupFile`](https://histoire.dev/reference/config.html#setupfile) in the config. You can see [examples here](https://github.com/histoire-dev/histoire/tree/main/examples).

**Reply by LouisAndrew** [reply]
Thanks a lot!
